### ISSUE TITLE
0.10.0: pytest is failing

### ISSUE THREAD

**Issue by kloczek** [novnc/websockify#498]
I'm trying to package your module as an rpm package. So I'm using the typical build, install and test cycle used on building packages from non-root account.
- "setup.py build"
- "setup.py install --root </install/prefix>"
- "pytest with PYTHONPATH pointing to sitearch and sitelib inside </install/prefix>

May I ask for help because one units are failing:
```console
+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-websockify-0.10.0~no_loopy_deps-2.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-websockify-0.10.0~no_loopy_deps-2.fc35.x86_64/usr/lib/python3.8/site-packages
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.1
rootdir: /home/tkloczko/rpmbuild/BUILD/websockify-0.10.0
plugins: forked-1.3.0, shutil-1.7.0, virtualenv-1.7.0, flake8-1.0.7, xdist-2.3.0, cov-2.12.1, mock-3.6.1
collected 57 items

tests/test_auth_plugins.py ....                                                                                                                                      [  7%]
tests/test_token_plugins.py ...F.......                                                                                                                              [ 26%]
tests/test_websocket.py .................                                                                                                                            [ 56%]
tests/test_websocketproxy.py .....                                                                                                                                   [ 64%]
tests/test_websockifyserver.py ....................                                                                                                                  [100%]

================================================================================= FAILURES =================================================================================
____________________________________________________________ JWSTokenTestCase.test_asymmetric_jwe_token_plugin _____________________________________________________________

self = <test_token_plugins.JWSTokenTestCase testMethod=test_asymmetric_jwe_token_plugin>

    def test_asymmetric_jwe_token_plugin(self):
        plugin = JWTTokenApi("./tests/fixtures/private.pem")

        private_key = jwt.JWK()
        public_key = jwt.JWK()
        private_key_data = open("./tests/fixtures/private.pem", "rb").read()
        public_key_data = open("./tests/fixtures/public.pem", "rb").read()
        private_key.import_from_pem(private_key_data)
        public_key.import_from_pem(public_key_data)
        jwt_token = jwt.JWT({"alg": "RS256"}, {'host': "remote_host", 'port': "remote_port"})
        jwt_token.make_signed_token(private_key)
        jwe_token = jwt.JWT(header={"alg": "RSA1_5", "enc": "A256CBC-HS512"},
                    claims=jwt_token.serialize())
>       jwe_token.make_encrypted_token(public_key)

tests/test_token_plugins.py:172:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/lib/python3.8/site-packages/jwcrypto/jwt.py:461: in make_encrypted_token
    t.add_recipient(key)
/usr/lib/python3.8/site-packages/jwcrypto/jwe.py:230: in add_recipient
    alg, enc = self._get_alg_enc_from_headers(jh)
/usr/lib/python3.8/site-packages/jwcrypto/jwe.py:183: in _get_alg_enc_from_headers
    alg = self._jwa_keymgmt(algname)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <jwcrypto.jwe.JWE object at 0x7f230e41d8e0>, name = 'RSA1_5'

    def _jwa_keymgmt(self, name):
        allowed = self._allowed_algs or default_allowed_algs
        if name not in allowed:
>           raise InvalidJWEOperation('Algorithm not allowed')
E           jwcrypto.common.InvalidJWEOperation: Algorithm not allowed

/usr/lib/python3.8/site-packages/jwcrypto/jwe.py:131: InvalidJWEOperation
============================================================================= warnings summary =============================================================================
../../BUILDROOT/python-websockify-0.10.0~no_loopy_deps-2.fc35.x86_64/usr/lib/python3.8/site-packages/websockify/websocket.py:31
  /home/tkloczko/rpmbuild/BUILDROOT/python-websockify-0.10.0~no_loopy_deps-2.fc35.x86_64/usr/lib/python3.8/site-packages/websockify/websocket.py:31: UserWarning: no 'numpy' module, HyBi protocol will be slower
    warnings.warn("no 'numpy' module, HyBi protocol will be slower")

-- Docs: https://docs.pytest.org/en/stable/warnings.html
========================================================================= short test summary info ==========================================================================
FAILED tests/test_token_plugins.py::JWSTokenTestCase::test_asymmetric_jwe_token_plugin - jwcrypto.common.InvalidJWEOperation: Algorithm not allowed
================================================================= 1 failed, 56 passed, 1 warning in 8.77s ==================================================================
```

**Comment by CendioOssman** [comment]
This was fixed in dc345815c0c344de115278a37e837ba6a6f1b272, which unfortunately isn't in 0.10.0. It will be in the next release though.

**Comment by kloczek** [comment]
Just tested that commit and indeed it sices the issue
```console
[tkloczko@ss-desktop SPECS]$ rpmbuild -ba --with check --with failing_tests python-websockify.spec
warning: Downloading https://github.com/kanaka/websockify//archive/v0.10.0/python-websockify-0.10.0.tar.gz to /home/tkloczko/rpmbuild/SOURCES/python-websockify-0.10.0.tar.gz
warning: Downloading https://github.com/kanaka/websockify//commit/dc345815.patch#/python-websockify-Use-RSA-OAEP-instead-of-RSA1_5-for-jwt-tests.patch to /home/tkloczko/rpmbuild/SOURCES/python-websockify-Use-RSA-OAEP-instead-of-RSA1_5-for-jwt-tests.patch
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.92nsHi
+ umask 022
+ cd /home/tkloczko/rpmbuild/BUILD
+ cd /home/tkloczko/rpmbuild/BUILD
+ rm -rf websockify-0.10.0
+ /usr/bin/gzip -dc /home/tkloczko/rpmbuild/SOURCES/python-websockify-0.10.0.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd websockify-0.10.0
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ /usr/bin/cat /home/tkloczko/rpmbuild/SOURCES/python-websockify-Use-RSA-OAEP-instead-of-RSA1_5-for-jwt-tests.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f
+ RPM_EC=0
++ jobs -p
+ exit 0

[..]

+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-websockify-0.10.0-2.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-websockify-0.10.0-2.fc35.x86_64/usr/lib/python3.8/site-packages
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.1
Using --randomly-seed=2301856818
rootdir: /home/tkloczko/rpmbuild/BUILD/websockify-0.10.0
plugins: forked-1.3.0, shutil-1.7.0, virtualenv-1.7.0, flake8-1.0.7, xdist-2.3.0, cov-2.12.1, mock-3.6.1, localserver-0.5.0, timeout-2.0.1, flaky-3.7.0, hypothesis-6.14.6, rerunfailures-9.1.1, checkdocs-2.7.1, anyio-3.3.1, tornasync-0.6.0.post2, randomly-3.8.0, freezegun-0.4.2
collected 58 items

tests/test_websockifyserver.py ....................                                                                                                                  [ 34%]
tests/test_token_plugins.py ...........                                                                                                                              [ 53%]
tests/test_websocketproxy.py .....                                                                                                                                   [ 62%]
. .                                                                                                                                                                  [ 63%]
tests/test_auth_plugins.py ....                                                                                                                                      [ 70%]
tests/test_websocket.py .................                                                                                                                            [100%]

=========================================================================== 58 passed in 13.78s ============================================================================
```

Thank you.
